man/systemd-system.conf.xml

```diff
@@ -361,8 +361,10 @@
         <listitem><para>Configure the default value for the per-unit <varname>TasksMax=</varname> setting. See
         <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry>
         for details. This setting applies to all unit types that support resource control settings, with the exception
-        of slice units. Defaults to 15%, which equals 4915 with the kernel's defaults on the host, but might be smaller
-        in OS containers.</para></listitem>
+        of slice units. Defaults to 15% of the kernel's option <varname>kernel.pid_max=</varname> or root cgroup <varname>"pids.max"</varname>.
+        Kernel has a default value for <varname>kernel.pid_max=</varname> and an algorithm of counting in case of more than 32 cores.
+        For example with default <varname>kernel.pid_max=</varname> <varname>DefaultTasksMax=</varname> will be equal to 4915,
```

Comment:
`default`?

At least, I think it is better to add a comma after `kernel.pid_max=`.

Reply:
Done.